lib/top_row.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'save_movie_button.dart';
+import 'stars.dart';
+import 'text_styles.dart';
+import 'like_counter.dart';
+import 'movie.dart';
+
+class TopRow extends StatelessWidget {
```

Comment:
Same behavior as i mentioned in the previous comment, review the parameters that you really need to use

Reply:
Done